azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/bindings/CustomBinding.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.maven.function.bindings;
+
+import java.lang.annotation.Annotation;
+
+public class CustomBinding extends Binding {
+
+    private com.microsoft.azure.functions.annotation.CustomBinding customBindingAnnotation;
```

Comment:
what's the reason for using full package name here? com.microsoft.azure.functions.annotation.CustomBinding

Reply:
This is the annotation class from azure function java library, in order not to mix it with CustomBinding Object, use the full name.